web/templates/inbox_list.hbs

```diff
@@ -0,0 +1,23 @@
+<div id="inbox_list" data-simplebar>
+    <div id="pm_header" tabindex="0" class="inbox_header {{#unless has_pms_post_filter}}hidden_by_filters{{/unless}}">
+        <div class="left_part_wrapper">
```

Comment:
Do we want to scope these names under the `inbox` namespace, e.g. `inbox-left-part-wrapper` and `#inbox-dm-header`? If they're not reusable elsewhere, I think it'll make it easier to reason about whether the selectors only apply to the inbox view or might do something weird too.

Reply:
Done.